### ISSUE TITLE
Cannot resolve module 'fs'

### ISSUE THREAD

**Issue by notiv-nt** [cfinke/Typo.js#42]
```
ERROR in ./~/typo-js/typo.js
Module not found: Error: Cannot resolve module 'fs' in C:\git\client\node_modules\typo-js
 @ ./~/typo-js/typo.js 164:12-25
```

Node v5.12.0
NPM 3.8.6
Win10

Using Webpack

**Comment by alassiter** [comment]
I'm also getting this with:

Node v5.3.0
NPM 3.10.6
and webpack

**Comment by benrlodge** [comment]
I'm running into the same issue with webpack running tests. @alassiter did you ever get this fixed?

I tried using
```
node: {
  fs: "empty"
}
```
but no luck

**Comment by noderat** [comment]
@cfinke Can a new release be pushed to npm with this commit included? npm still reflects the prior package.json.

**Comment by cfinke** [comment]
Sorry about that, I've published version 1.0.2 on npm now.
